README.md

```diff
@@ -17,28 +16,20 @@ This is the bot that powers the Discord Feedback server, develop'd by the volunt
 
 ## Development
 
-We here at Discord Feedback Bot Tinkerers Sqaud™ use Visual Studio Code, with the [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), [npm](https://marketplace.visualstudio.com/items?itemName=eg2.vscode-npm-script), [npm Intellisense](https://marketplace.visualstudio.com/items?itemName=christian-kohler.npm-intellisense) plugins in order to produce the bestest code.
+You can find code style & other details for contributing (such as tools we use) in the [CONTRIBUTING.MD](DiscordFeedback/.github/CONTRIBUTING.md)
```

Comment:
Since you're referencing things that aren't in stable yet, your PR will most likely be on hold until this is released to stable.

Or you could change your base branch :shipit: